eZ/Publish/Core/Persistence/Legacy/Content/Gateway/DoctrineDatabase/QueryBuilder.php

```diff
@@ -133,7 +133,7 @@ public function createFindQuery( array $translations = null )
             )
         );
 
-        if ( $translations !== null )
+        if ( $translations )
```

Comment:
would be easier to read(, and probably faster, but that does not matter so much in this context) to write it like so:
```if ( !empty( $translations ) )```